config/Config-images.in

```diff
@@ -16,6 +16,7 @@ menu "Target Images"
 			prompt "Compression"
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_apm821xx
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_ar71xx
+			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_ath79
```

Comment:
@ch6574 just to make sure, last question: can you boot a 4.19 initramfs **without** this option?

Reply:
It hasn't worked for me, so "no".

I can test again later today to 100% confirm though.

Reply:
Tested again, and it will not.

Image built was `uncompressed size: 12879652 bytes`